examples/mpi-mnist/mpi_backend/gpu/mpi_mnist_job_gpu.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: "kubeflow.org/v1alpha2"
+kind: "PyTorchJob"
+metadata:
+  name: "pytorch-mpi-mnist-gpu"
+spec:
+  pytorchReplicaSpecs:
+    Master:
+      replicas: 1
+      restartPolicy: OnFailure
+      template:
+        spec:
+          containers:
+            - name: pytorch
+              image: docker.io/akado2009/pytorch-mpi-mnist-gpu
+    Worker:
+      replicas: 3
+      restartPolicy: OnFailure
+      template:
+        spec:
+          containers:
+            - name: pytorch
+              image: docker.io/akado2009/pytorch-mpi-mnist-gpu
```

Comment:
Please add a new line at the end of the file.

Reply:
Fixed.